### ISSUE TITLE
[KafkaSpinner] Test on clean Ubuntu 12.04 and Ubuntu 14.04 Digital ocean box

### ISSUE THREAD

**Issue by jeroldleslie** [DemandCube/NeverwinterDP_Legacy#382]

**Comment by jeroldleslie** [comment]
@KingOfPoptart test on ubuntu 12.04 and ubuntu 14.04 with root user. Not working with other user since docker daemon always runs as the root user. It may be fixed later.

**Comment by KingOfPoptart** [comment]
Looks like its working!  Good work, Peter - it'll be nice to see this running without having to use the root user, but lets save that for an enhancement later.

I'm having trouble connecting to zookeeper?  its unclear as to what machine is what in the output

```
Updating /etc/hosts on all kafka nodes...
Warning: Permanently added '[ubuntu14]:49195' (ECDSA) to the list of known hosts.
Warning: Permanently added '[ubuntu14]:49196' (ECDSA) to the list of known hosts.
Warning: Permanently added '[ubuntu14]:49197' (ECDSA) to the list of known hosts.
Warning: Permanently added '[ubuntu14]:49198' (ECDSA) to the list of known hosts.
Warning: Permanently added '[ubuntu14]:49199' (ECDSA) to the list of known hosts.
Warning: Permanently added '[ubuntu14]:49200' (ECDSA) to the list of known hosts.
Start zookeeper in all node
```

Can we clean up the output?

```
Node failure will occur in 101 seconds.
Node failure will occur in 100 seconds.
...
```

Is noisy and hard to follow.  I think a single message of "Node will fail in X seconds" is plenty - it doesn't need to refresh every second.

Also it would be helpful to output the host:port of all the nodes - i.e. something like

```
knode1 - localhost:12345
knode2 - localhost:12346
zoo1 - localhost:12347
zoo2 - localhost:12348
```
